model/cluster_request.go

```diff
@@ -17,6 +17,8 @@ const (
 	NetworkingCalico = "calico"
 	// NetworkingAmazon is Amazon networking plugin.
 	NetworkingAmazon = "amazon-vpc-routed-eni"
+	// NetworkingVpcCni is Amazon VPC CNI networking plugin.
+	NetworkingVpcCni = "vpc-cni"
```

Comment:
```suggestion
	NetworkingVpcCni = "amazon-vpc-cni"
```

I think this would be a bit clearer for anyone reviewing cluster configuration. Just want to be sure that there is no confusion about the nature of the CNI.